src/modules/market/selectors/market.js

```diff
@@ -291,7 +291,7 @@ export function assembleMarket(
 
       market.tags = (market.tags || []).map((tag) => {
         const obj = {
-          name: tag && tag.toString().toLowerCase().trim(),
+          name: tag && tag.toString().trim(),
```

Comment:
Sounds good!  

Yeah, it shouldn’t impact the ultimate display of the market; however, would we still want to trim w/in the components (All Tags, Tags in market preview, etc.), so we don’t have odd leading/trailing whitespace?